src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/FlowValidationHubBolt.java

```diff
@@ -62,7 +68,24 @@ public FlowValidationHubBolt(Config config, PersistenceManager persistenceManage
     @Override
     public void prepare(Map stormConf, TopologyContext context, OutputCollector collector) {
         super.prepare(stormConf, context, collector);
-        service = new FlowValidationHubService(persistenceManager, flowResourcesConfig);
+        service = new FlowValidationHubService(persistenceManager, flowResourcesConfig,
+                new FlowValidationHubCarrierImpl(null));
```

Comment:
Could we introduce a default constructor instead of passing null? Currently, it's ambiguous and not clear why null.